src/clusterfuzz/_internal/protos/swarming.proto

```diff
@@ -0,0 +1,1177 @@
+// Copyright 2022 The LUCI Authors. All rights reserved.
```

Comment:
copied over from luci code, don't know what the procedure is here wrt copyright and license?
@jonathanmetzman

Reply:
Probably needs to go in protos/third_party/.
Also, how will this be kept up to date?

Reply:
I talked to swarming folks, they don't expect this will change, and even if it changes, changes will be backward compatible, meaning we wouldn't need to update this file (at least not often).
I trimmed down the file to only the parts necessary for a NewTaskRequest message.

Reply:
I see. Can we add a monitoring metric (maybe in a followup PR) to record the rate of successful and unsuccessful task pushes? It would be good to know in real time if we start to fail to schedule tasks due to a change in the swarming API, for example.

Reply:
Yeah I can add that in a follow up pr!